mp-prod/conf.d/health_metrics.yaml

```diff
@@ -712,3 +712,17 @@ health_metrics:
         weight: 1
       - expression: "community.api_down"
         weight: 2
+  ### FunctionGraph
+  functiongraph:
+    service: functiongraph
+    component_name: "FunctionGraph"
+    category: compute
+    metrics:
+      - functiongraph.api_slow
+      - functiongraph.api_down
+      - functiongraph.api_success_rate_low
+    expressions:
+      - expression: "functiongraph.api_slow || functiongraph.api_success_rate_low || functiongraph.api_slow || functiongraph.api_success_rate_low"
```

Comment:
why doubling?

Reply:
Because I copy-pasted from cts, where v2 and v3 exists, and I did not realize the duplication.